go/libkb/util.go

```diff
@@ -1034,8 +1034,8 @@ var kbfsPathInnerRegExp = func() *regexp.Regexp {
 	// e.g. alice@twitter
 	const regularAssertion = `[-_a-zA-Z0-9.+]+@[a-zA-Z.]+`
 	// e.g. [bob@keybase.io]@email
-	const atContainingAssertion = `\[[-_a-zA-Z0-9.]+@[-_a-zA-Z0-9.]+\]@[a-zA-Z.]+`
-	const socialAssertion = `(?:` + regularAssertion + `)|(?:` + atContainingAssertion + `)`
+	const emailAssertion = `\[[-_+a-zA-Z0-9.]+@[-_a-zA-Z0-9.]+\]@[a-zA-Z.]+`
```

Comment:
I could be mistaken, but don't we need a `\` before the `+`?  Otherwise this means one or more underscores?

Reply:
I think the fact that it's inside `[]` removes the need for `/`. I tested in chat to make sure it worked, but will also add some tests for this.

Reply:
Cool thanks.  That's confusing.  Stupid regexes.

Reply:
😆